### PR TITLE
Fix jestSupport preprocessor for babel-core location

### PR DESCRIPTION
**Pull request by adrienthiery**
Proposition to fix issue #3999 

All feedback appreciated, this is my first Pull request on a big project. :)

### CODE REVIEW

File: jestSupport/preprocessor.js [hunk 1]
```diff
@@ -11,6 +11,15 @@
 const createCacheKeyFunction = require('fbjs-scripts/jest/createCacheKeyFunction');
 const path = require('path');
 const transformer = require('../packager/transformer.js');
+const fs = require('fs');
+
+var babel_core_location = path.join(__dirname, '../node_modules/babel-core/package.json');
```
Comment (satya164) [inline jestSupport/preprocessor.js]: Can you change this to camel case please?

`babelCoreLocation`

File: jestSupport/preprocessor.js [hunk 2]
```diff
@@ -11,6 +11,15 @@
 const createCacheKeyFunction = require('fbjs-scripts/jest/createCacheKeyFunction');
 const path = require('path');
 const transformer = require('../packager/transformer.js');
+const fs = require('fs');
+
+var babelCoreLocation = path.join(__dirname, '../node_modules/babel-core/package.json');
+try {
+  fs.lstatSync(babelCoreLocation);
+}
+catch(e) {
```
Comment (satya164) [inline jestSupport/preprocessor.js]: nit: move `catch` to the same line as `}`

**Comment by facebook-github-bot-1** [comment]
@adrienthiery do you have any updates for this pull request? It's been a while so we wanted to check in and see if you've looked at the requested changes.

**Comment by jsierles** [comment]
This appears to have been resolved by a3706411ab5c8e0d49ab3c1930d223fa40d38a7e. Thanks!

**Pull request closed without merging**